libs/BehaviorKit/tests/BehaviorKit_test.cpp

```diff
@@ -68,7 +68,7 @@ TEST_F(BehaviorKitTest, launching)
 TEST_F(BehaviorKitTest, sleeping)
 {
 	EXPECT_CALL(mock_videokit, playVideoOnce);
-	EXPECT_CALL(mock_ledkit, start(&led::animation::sleeping)).Times(1);
+	EXPECT_CALL(mock_ledkit, start).Times(1);
```

Comment:
Best to keep the previous test, explicit what kind of animation is expected on sleeping

Reply:
I know there is some trouble to complete this test in that condition, I am checking on it

Reply:
You probably should reversed this commit

Reply:
Fixed with [5d61435](https://github.com/leka/LekaOS/pull/1206/commits/5d6143500dd524af356500c6bac86a0a2e1b39e1)